tasks/IKF.cpp

```diff
@@ -98,6 +98,9 @@ void IKF::imu_samplesTransformerCallback(const base::Time &ts, const ::base::sam
             if (correction_idx == correction_numbers)
             {
                 acc = correctionAcc / (double)correction_numbers;
+
+                acc_body = acc; // save this to be output later after being corrected
```

Comment:
The intent was that the acceleration output occurs at the same rate as the orientation output, for simplicity, but we could output every corrected sample as well if this can be handled.